app/controllers/forms_controller.rb

```diff
@@ -131,6 +131,8 @@ def update
       @form.assign_attributes(form_params)
       authorize!(:rename, @form) if @form.name_changed?
       @form.save!
+      ODK::FormRenderJob.perform_later(@form) if @form.live?
+
       if params[:save_and_go_live].present?
         @form.update_status(:live)
```

Comment:
looks like it can also be marked live after the point where we're checking if it's live, i'll make a tweak to address this

Reply:
ah, but setting live causes it to render. so i think this is fine, just confusing to me, i added a comment to clarify.